deprecated.tex

```diff
@@ -103,3 +98,109 @@
     transform(Rng&& rng1, I first2, O result,
               F binary_op, Proj1 proj1 = Proj1{}, Proj2 proj2 = Proj2{});
 \end{codeblock}
+
+\rSec2[depr.algo.mismatch]{Mismatch}
```

Comment:
I think we can get away without all of these subsections if we add a single *Remarks* element like "These signatures behave as described in Clause~\ref{algorithms} with the name `last2` defined as `first2 + (last1 - first1)`."